jujugui/static/gui/src/app/components/deployment-flow/payment/payment.js

```diff
@@ -89,8 +93,252 @@ YUI.add('deployment-payment', function() {
+            </h2>
+            <juju.components.GenericInput
+              disabled={disabled}
+              label="Card number"
+              required={true} />
+            <div className="twelve-col">
+              <div className="six-col">
+                <juju.components.GenericInput
+                  disabled={disabled}
+                  label="Expiry MM/YY"
+                  required={true} />
+              </div>
+              <div className="six-col last-col">
+                <juju.components.GenericInput
+                  disabled={disabled}
+                  label="Security number (CVC)"
+                  required={true} />
+              </div>
+            </div>
+            <div className="twelve-col">
+              <juju.components.GenericInput
+                disabled={disabled}
+                label="Name on card"
+                required={true} />
+            </div>
+            <label htmlFor="cardAddressSame">
+              <input checked={this.state.cardAddressSame}
+                id="cardAddressSame"
+                name="cardAddressSame"
+                onChange={this._handleCardSameChange}
+                type="checkbox" />
+              Credit or debit card address is the same as above
+            </label>
+            <label htmlFor="billingAddressSame">
+              <input checked={this.state.billingAddressSame}
+                id="billingAddressSame"
+                name="billingAddressSame"
+                onChange={this._handleBillingSameChange}
+                type="checkbox" />
+              Billing address is the same as above
+            </label>
+            {this._generateCardAddressFields()}
+            {this._generateBillingAddressFields()}
+          </div>
+          <div className="deployment-payment__add">
+          <juju.components.GenericButton
+            action={null}
+            disabled={disabled}
+            type="inline-neutral"
+            title="Add payment details" />
+          </div>
+        </form>);
+    },
+
+    /**
+      Generate the fields for an address.
+
+      @method _generateAddressFields
+    */
+    _generateAddressFields: function() {
+      const disabled = this.props.acl.isReadOnly();
+      return (
+        <div>
+          <juju.components.InsetSelect
+            disabled={disabled}
+            label="Country"
+            onChange={null}
+            options={[]} />
+          <juju.components.GenericInput
+            disabled={disabled}
+            label="Full name"
+            required={true} />
+          <juju.components.GenericInput
+            disabled={disabled}
+            label="Address line 1"
+            required={true} />
+          <juju.components.GenericInput
+            disabled={disabled}
+            label="Address line 2 (optional)"
+            required={false} />
+          <juju.components.GenericInput
+            disabled={disabled}
+            label="County (optional)"
```

Comment:
I think calling it State/Province might be less confusing, I had so many confusion in the payment service.